src/org/labkey/filetransfer/view/fileTransfer.jsp

```diff
@@ -32,12 +32,24 @@
     public void addClientDependencies(ClientDependencies dependencies)
     {
         dependencies.add("Ext4");
+        dependencies.add("tsstatic/css/filetransfermodule.css");
```

Comment:
This file is in a completely different (trialshare-specific) project, so it will not be included in the files I just pushed.  As long as the page calls for this file, we'll worry about actually providing it.

Reply:
This will cause errors for anyone who doesn't have your trialShare-specific files, so you should really put a file in the resources directory for this module that defines the CSS classes you need.

Reply:
Ah, naturally.  I think there may be some confusion about whether this is meant for generic use; we should discuss it when Dennis is back.

Reply:
Our concern there was being able to update the .css on the fly without having to swap out the fileTransfer.module.  TS static is an exploded folder in external modules, so we don't have to worry about server shutdown to change it, or risking putting a temp version in the exploded ftm module but then having it overwritten by the .module if the server does restart.  We'll figure out a way to include it on git.